BlockServer/core/inactive_config_holder.py

```diff
@@ -13,45 +13,91 @@
 # along with this program; if not, you can obtain a copy from
 # https://www.eclipse.org/org/documents/epl-v10.php or
 # http://opensource.org/licenses/eclipse-1.0.php
-
-import json
-
 from BlockServer.core.config_holder import ConfigHolder
 
 
 class InactiveConfigHolder(ConfigHolder):
-    """ Class to hold a individual inactive configuration or component.
     """
-    def __init__(self, macros, file_manager):
-        """ Constructor.
+    Class to hold a individual inactive configuration or component.
+    """
+    def __init__(self, macros, file_manager, test_config=None):
```

Comment:
This would affect a lot of tests and I don't think it's really in scope for this ticket, I've added a new ticket https://github.com/ISISComputingGroup/IBEX/issues/4315